SpeechResponder/ViewScriptWindow.xaml

```diff
@@ -25,7 +26,16 @@
             <TextBlock Margin="10" Grid.Row="0" Grid.Column="1" Text="{Binding Path=ScriptName}"/>
             <TextBlock Margin="10" Grid.Row="1" Grid.Column="0" Text="{x:Static resx:SpeechResponder.header_description}" />
             <TextBlock Margin="10" Grid.Row="1" Grid.Column="1" Text="{Binding Path=ScriptDescription}"/>
```

Comment:
Consider using `Margin="10, 5, 10, 0"` for lines 25-28 (more compact).

Reply:
Done